website/integrations/services/home-assistant/index.md

```diff
@@ -13,54 +13,57 @@ title: Home-Assistant
-
-Afterwards, make sure the `trusted_proxies` setting contains the IP(s) of the Host(s) authentik is running on.
-
-Use this configuration to match on the user's authentik username.
-
-```yaml
-auth_header:
-    username_header: X-authentik-username
-```
-
-If this is not the case, you can simply add an additional header for your user, which contains the Home-Assistant Name and authenticate based on that.
-
-For example add this to your user's properties and set the Header to `X-ak-hass-user`.
-
-```yaml
-additionalHeaders:
-    X-ak-hass-user: some other name
-```
-
-## authentik
-
-Create a Proxy Provider with the following values
-
--   Internal host
-
-    If Home-Assistant is running in docker, and you're deploying the authentik proxy on the same host, set the value to `http://homeassistant:8123`, where Home-Assistant is the name of your container.
-
-    If Home-Assistant is running on a different server than where you are deploying the authentik proxy, set the value to `http://hass.company:8123`.
-
--   External host
-
-    Set this to the external URL you will be accessing Home-Assistant from.
-
-Create an application in authentik and select the provider you've created above.
-
-## Deployment
-
-Create an outpost deployment for the provider you've created above, as described [here](../../../docs/outposts/). Deploy this Outpost either on the same host or a different host that can access Home-Assistant.
-
-The outpost will connect to authentik and configure itself.
+## Home Assistant configuration
+
+1. Configure [trusted_proxies](https://www.home-assistant.io/integrations/http/#trusted_proxies) for the HTTP integration with the IP(s) of the Host(s) authentik is running on.
+2. If you don't already have it set up, https://github.com/BeryJu/hass-auth-header, using the installation guide.
+3. There are two ways to configure the custom component.
+    1. To match on the user's authentik username, use the following configuration:
+        ```yaml
+        auth_header:
+            username_header: X-authentik-username
+        ```
+    2. Alternatively, you can associate an existing Home Assistant username to an authentik username.
+        1. Within authentik, navigate to **Directory** > **Users**.
+        2. Select **Edit** for the user then add the following configuration to the **Attributes** section. Be sure to replace `hassusername` with the Home Assistant username.
+           :::note
+           This configuration will add an additional header for the authentik user which will contain the Home Assistant username and allow Home Assistant to authenticate based on that.
+           :::
+            ```yaml
+            additionalHeaders:
+                X-ak-hass-user: hassusername
+            ```
+        3. Then configure the Home Assistant custom component to use this header:
+            ```yaml
+            auth_header:
+                username_header: X-ak-hass-user
+            ```
+
+## authentik configuration
+
```

Comment:
Let's move the authentik config section to be first, so that it is consistent with [the majority] of the other integrations. Basically, let's keep this order:
What is XYZ
Preparation
authentik configuration
XYZ configuration

Reply:
Agreed, good spot! Sorted now 😄